conveyor/slot.go

```diff
@@ -148,7 +167,7 @@ func initElementsBuf() ([]slotElement, *ElementList) {
 }
 
 // NewSlot creates new instance of Slot
-func NewSlot(pulseState constant.PulseState, pulseNumber core.PulseNumber, removeSlotCallback RemoveSlotCallback) *Slot {
+func NewSlot(pulseState constant.PulseState, pulseNumber core.PulseNumber, removeSlotCallback RemoveSlotCallback, runWorker bool) *Slot {
```

Comment:
you do not need flag runWorker, you can define another constructor in test scope

Reply:
you can separate constructing of slot object into another function (without starting worker gorutine) and use new function in tests